celery_serverless/handler.py

```diff
@@ -94,7 +94,11 @@ def worker(event, context):
         _maybe_call_hook(_error_handler_call_envvar, locals())
         raise
     finally:
-        logger.info('END: Handle request ID: %s', request_id)
+        try:
+            logger.info('END: Handle request ID: %s', request_id)
+        except:
```

Comment:
Maybe define `request_id` as `None` at be beginning and avoid a possible `NameError`.